tests/fafnir_tests/mod.rs

```diff
@@ -102,8 +103,41 @@ fn load_poi_class_function(conn: &Connection) {
             $$ LANGUAGE SQL IMMUTABLE;", &[]).unwrap();
 }
 
-pub fn main_test(es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapper) {
-    init(&pg_wrapper);
+fn load_address(es_wrapper: &mut ElasticSearchWrapper) {
+    let test_address = &get_test_address();
+    es_wrapper.make_addr_index("test", test_address);
```

Comment:
shouldn't the dataset be a global variable ?

Reply:
done.